src/braket/aws/aws_session.py

```diff
@@ -72,9 +72,18 @@ def __init__(
                 f"Braket Client region is '{braket_client.meta.region_name}'."
             )
 
-        self._config = config
+        self._update_user_agent()
+        self._config = Config(user_agent_extra=self._braket_user_agents)
+        if config:
+            self._config = self._config.merge(config)
+        if braket_client and braket_client._client_config:
+            self._config = self._config.merge(braket_client._client_config)
```

Comment:
Should this be moved into the `if braket_client` branch?

Reply:
yes, makes sense